zeebe-cluster/templates/configmap.yaml

```diff
@@ -20,38 +24,14 @@ data:
     export ZEEBE_CONTACT_POINTS="${ZEEBE_CONTACT_POINTS}"
     
     exec /usr/local/zeebe/bin/broker
-  zeebe.cfg.toml: |
+ {{- end }}
+ {{- if .Values.zeebeConfig }}
+  zeebe.cfg.toml: |-
+{{ .Values.zeebeConfig | nindent 4 }}
+ {{- else }}
+  zeebe.cfg.toml: |-
     # For more information about this configuration visit: https://docs.zeebe.io/operations/the-zeebecfgtoml-file.html
     [threads]
-    cpuThreadCount = {{ .Values.cpuThreadCount | quote }}
```

Comment:
@ysaakpr I am guessing that here you are interested in providing your own configuration file.. shouldn't we just parameterise the entire ConfigMap? so basically if you want to override the default CM you just can set the name of a custom CM? that will allow you to configure whatever parameter that you want. 
With the Default CM we should keep a default configuration plus some externalised parameters as the one that are provided today.

Reply:
If you want to maintain two configs one, with externalized parameters, another raw config, which can eventually override the default CM, yeah we can try to do that.

Reply:
How to deal with multiple config files with respect to zeebe, concatenate the files  ??

Reply:
@ysaakpr just one configMap at the time.. no need for multiple, but when we select which ConfigMap to use we expose a Value to choose which one.. the provided by default or the one provided by the user.. does that make sense? Can you implement that change?

Reply:
Look here: https://github.com/zeebe-io/zeebe-cluster-helm/blob/d90e8603f9a4ea78407aa582c0dc3085ee19fe14/zeebe-cluster/templates/statefulset.yaml#L72